exporter.go

```diff
@@ -488,12 +500,29 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 	ch <- prometheus.MustNewConstMetric(clusterHealthRelocatingShardsDesc, prometheus.GaugeValue, float64(clusterHealth.RelocatingShards), clusterHealth.ClusterName)
 	ch <- prometheus.MustNewConstMetric(clusterHealthUnassignedShardsDesc, prometheus.GaugeValue, float64(clusterHealth.UnassignedShards), clusterHealth.ClusterName)
 
+	healthStatus := 0.0
 	statusIsGreen := 0.0
 	if clusterHealth.Status == "green" {
```

Comment:
I'd replace the ifs by an switch statement and remove the status_is_* metric altogether.